core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/TextInteractionDirective.js

```diff
@@ -0,0 +1,140 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for a text interaction question in the simple editor.
+ */
+
+// NOTE TO DEVELOPERS: This is meant to be a reusable directive, so its only
+// dependencies should be standard utility services. It should not have any
+// concept of "state in an exploration".
+
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
Looks like this section is repeated, remove.

Reply:
Done.